core/src/main/java/org/elasticsearch/script/ScriptParameterParser.java

```diff
@@ -202,7 +208,12 @@ public ScriptParameterValue getDefaultScriptParameterValue() {
     }
 
     public ScriptParameterValue getScriptParameterValue(String parameterName) {
-        return parameterValues.get(parameterName);
+        ScriptParameterValue scriptParameterValue = parameterValues.get(parameterName);
+        if (scriptParameterValue != null) {
+            deprecationLogger.deprecated("this from of the script api is deprecated and will be removed from the java api in the next "
```

Comment:
May be my lack of english. Do you mean "This method from the script API is deprecated"?